resources/views/network/internet/wifi_password.blade.php

```diff
@@ -38,3 +41,23 @@
         </div>
     </div>
 </div>
+
+<script>
+document.getElementById("pw").addEventListener("click", show_button);
```

Comment:
Don't we have to add these in a `$(document).ready` function?

Reply:
You could also add the function below there, we don't have to name it as it's used only once.